src/main/java/uk/gov/hmcts/juror/api/moj/domain/JurorPool.java

```diff
@@ -136,6 +136,9 @@ public class JurorPool implements Serializable {
     @Column(name = "date_created")
     private LocalDateTime dateCreated;
 
+    @Column(name = "response_entered")
+    private Boolean responseEntered;
+
```

Comment:
This should be on the Juror record not JurorPool record

Reply:
updated